### PR TITLE
feat(docs): framework-less guide

### PR DESCRIPTION
**Pull request by twisttaan**
Implements a guide on how to use Disploy with Deno and Node.js without the framework supplied from the CLI.

### CODE REVIEW

File: apps/docs/docs/Reference/framework-less/index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 1
+---
+
+# Using Disploy without the CLI
+
+Using the Disploy CLI is completely optional, though reccomended. It's purpose is to bundle your bot into a single file that can be deployed to Cloudflare Workers or imported into your own project. If you don't want to use the CLI, you can still use Disploy, but you'll have to bundle your bot yourself.
```
Comment (GodderE2D) [inline apps/docs/docs/Reference/framework-less/index.md:7]:
```suggestion
Using the Disploy CLI is completely optional, though recommended. Its purpose is to bundle your bot into a single file that can be deployed to Cloudflare Workers or imported into your own project. If you don't want to use the CLI, you can still use Disploy, but you'll have to bundle your bot yourself.
```

Grammatical/spelling errors

File: apps/docs/docs/Reference/framework-less/starting.mdx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+Once you have downloaded ngrok, you can run the following command to expose your bot to the internet:
+
+```bash
+ngrok http 8000
+```
+
+Once ngrok is running, you should do the following:
+
+1. Visit https://discord.com/developers/applications/your_bot_id/information
+2. Set INTERACTIONS ENDPOINT URL to https://xxxx-xx-xxx-xxx-xx.ngrok.io/interactions
```
Comment (GodderE2D) [inline apps/docs/docs/Reference/framework-less/starting.mdx:123]: Perhaps add an image?

File: apps/docs/docs/Reference/framework-less/creating-main-file.mdx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+await loadEnv({
+	export: true,
+});
+```
+
+</TabItem>
+
+<TabItem value="Node" label="Node">
+
```
Comment (twisttaan) [inline apps/docs/docs/Reference/framework-less/creating-main-file.mdx:29]: ditto but instead explain to install deps with npm, yarn, etc

**Review by twisttaan — COMMENTED** [review]
so i remember tmrw morning

File: apps/docs/docs/Reference/framework-less/starting.mdx [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+server.use(bodyParser.json());
+
+server.post('/interactions', async (req, res) => {
+	const tReq: TRequest = {
+		body: req.body,
+		headers: req.headers,
+		_request: req,
+	};
+
+	const payload = await app.router.entry(tReq);
```
Comment (GodderE2D) [inline apps/docs/docs/Reference/framework-less/starting.mdx:91]: Ditto

File: apps/docs/docs/Reference/framework-less/creating-main-file.mdx [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+<Tabs>
+
+<TabItem value="Deno" label="Deno">
+
+```js
+const { clientId, token, publicKey } = {
+	clientId: Deno.env.get('DISCORD_CLIENT_ID'),
+	token: Deno.env.get('DISCORD_TOKEN'),
+	publicKey: Deno.env.get('DISCORD_PUBLIC_KEY'),
+};
```
[the remark this answers is not in the document]
Comment (GodderE2D) [reply apps/docs/docs/Reference/framework-less/creating-main-file.mdx:51]: I don't know what an import map is but if that is more beneficial you can use that.

File: apps/docs/docs/Reference/framework-less/creating-main-file.mdx [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+}
+```
+
+## Breaking down the code
+
+We start by importing the `App` class from Disploy. We also import the `loadEnv` function from `std/dotenv/mod.ts` (Deno) or `dotenv` (Node.js). This function will load our environment variables from a `.env` file.
+
+We then create a new instance of the `App` class and export it so we can use it in other files. We then call the `start` method on the `app` instance and pass in our client ID, token and public key.
+
+We then loop through our commands and register them with the `app.commands.registerCommand` method. We also set the `debug` option to `true` in the `logger` option of the `App` class. This helps us debug our bot.
```
[the remark this answers is not in the document]
Comment (twisttaan) [reply apps/docs/docs/Reference/framework-less/creating-main-file.mdx:98]: would also be hard to maintain

File: apps/docs/docs/Reference/framework-less/creating-main-file.mdx [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+	export: true,
+});
+```
+
+</TabItem>
+
+<TabItem value="Node" label="Node">
+
+```js
+import 'dotenv/config';
```
[the remark this answers is not in the document]
Comment (twisttaan) [reply apps/docs/docs/Reference/framework-less/creating-main-file.mdx:31]: duplicate of https://github.com/Disploy/disploy/pull/62#discussion_r1022809896

File: apps/docs/docs/Reference/framework-less/creating-main-file.mdx (hunk 3 above)
Comment (twisttaan) [reply apps/docs/docs/Reference/framework-less/creating-main-file.mdx:29]: resolved in https://github.com/Disploy/disploy/pull/62/commits/92e7fa9f743a72e97d645e70e7dffc8a3e277527

File: apps/docs/docs/Reference/framework-less/creating-main-file.mdx [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+import TabItem from '@theme/TabItem';
+
+# Creating the main file
+
+Firstly open your code editor and create a new file called `main.mjs` in a folder named "src" of your project. This is where we'll be writing our bot's code.
+
+<Tabs>
+
+<TabItem value="Deno" label="Deno">
+
```
[the remark this answers is not in the document]
Comment (twisttaan) [reply apps/docs/docs/Reference/framework-less/creating-main-file.mdx:15]: resolved in https://github.com/Disploy/disploy/pull/62/commits/92e7fa9f743a72e97d645e70e7dffc8a3e277527

File: apps/docs/docs/Reference/framework-less/starting.mdx (hunk 4 above)
Comment (twisttaan) [reply apps/docs/docs/Reference/framework-less/starting.mdx:139]: resolved in https://github.com/Disploy/disploy/pull/62/commits/61289367cd1a74ecd13e028cc94894a0fcdfff6e

File: apps/docs/docs/Reference/framework-less/starting.mdx [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+			};
+			for (const [key, value] of Object.entries(headers)) {
+				if (typeof value === 'string') {
+					headersObj[key] = value;
+				}
+				if (Array.isArray(value)) {
+					headersObj[key] = value.join(',');
+				}
+			}
+			return headersObj;
```
[the remark this answers is not in the document]
Comment (twisttaan) [reply apps/docs/docs/Reference/framework-less/starting.mdx]: resolved in https://github.com/Disploy/disploy/pull/62/commits/61289367cd1a74ecd13e028cc94894a0fcdfff6e

File: apps/docs/docs/Reference/framework-less/starting.mdx (hunk 2 above)
Comment (twisttaan) [reply apps/docs/docs/Reference/framework-less/starting.mdx:201]: resolved in https://github.com/Disploy/disploy/pull/62/commits/f7563ba284fce8ef1c28c721504692e7a3b8242e

File: apps/docs/docs/Reference/framework-less/creating-main-file.mdx [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+	token: process.env.DISCORD_TOKEN,
+	publicKey: process.env.DISCORD_PUBLIC_KEY,
+};
+```
+
+</TabItem>
+
+</Tabs>
+
+We then create a new instance of the `App` class and export it so we can use it in other files. We then call the `start` method on the `app` instance and pass in our client ID, token and public key.
```
[the remark this answers is not in the document]
Comment (twisttaan) [reply apps/docs/docs/Reference/framework-less/creating-main-file.mdx]: resolved in https://github.com/Disploy/disploy/pull/62/commits/8d9b7695e7df20957a30f0442335fba6738c59ea

**Pull request merged**